CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v0.7.0 (Unreleased)
+### DEPRECATED
```

Comment:
imo we don't need the empty section headers

Reply:
I debated about this with myself.  Easy enough to add back.  Done.